de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/docs/sales/ordercheckup_with_barcode/report.jrxml

```diff
@@ -6,7 +6,7 @@
 	<property name="ireport.zoom" value="1.100000000000001"/>
 	<property name="ireport.x" value="0"/>
 	<property name="ireport.y" value="0"/>
-	<property name="com.jaspersoft.studio.data.defaultdataadapter" value="metasfresh"/>
+	<property name="com.jaspersoft.studio.data.defaultdataadapter" value="mf15"/>
```

Comment:
wondering why using `mf15` instead of `metasfresh`.
I think `metasfresh` is much more intuitive for new people.
wdyt?

Reply:
done